dev-lang/luajit/luajit-2.0.4.ebuild

```diff
@@ -42,6 +43,7 @@ _emake() {
 		DYNAMIC_CC="$(tc-getCC) -fPIC" \
 		TARGET_LD="$(tc-getCC)" \
 		TARGET_AR="$(tc-getAR) rcus" \
+		BUILDMODE="$(usex static-libs mixed dynamic)" \
 		TARGET_STRIP="true" \
 		INSTALL_LIB="${ED%/}/usr/$(get_libdir)" \
 		"$@"
```

Comment:
src_install: `dohtml` is deprecated in `EAPI 6`, either use `docinto html; dodoc -r .` or `HTML_DOCS=()` + `einstalldocs` (I prefer the latter)
Also, `|| die` missing on cd